pkg/components/util/install.go

```diff
@@ -29,31 +29,38 @@ import (
 	"github.com/kinvolk/lokomotive/pkg/k8sutil"
```

Comment:
Line #1 of commit message is missing some commas or double quotes because it sounds weird and hard to understand unless someone looks at the code.

Reply:
I changed the commit message to clarify. Hopefully makes more sense now.